packages/neptune-extension/src/nbextension/Toolbar.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+import {
+  injectNotebookId,
+  getNotebookId,
+} from 'platform/meta';
+
+const NEPTUNE_LOGO_URL = 'url(\'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA4AAAANCAYAAACZ3F9/AAAABHNCSVQICAgIfAhkiAAAAN5JREFUKJHF0K9Lw0EcxvGXc07ZEJTJBIOrBothJsHmLAOLMLQJ/lNGu0E2ozKDJoMIgqIOy0wiIiZ/MIOfLxzD7B74cHfvu+eezx3DUg4HOEMx4WM4xgXGBw1QxxIm0Ej2N1CJy9ZT40iMe3jFE5axFXwfN/jCAnYGW73GJlbQS/g91rCKh7/e+IJqtPSJOUyjjxlM4hvlqMt8GOfxHvMr1GLdxXPw2+B9TI0G/EjSFyOljDe0gtf8ft5sdv4I7cTYwGMkNBO+jbv4j104jMqUwzlOIyFTASfooOTf9QMT5SpplgZyIQAAAABJRU5ErkJggg==\')';
+
+function initializeToolbar(callback1 = () => {}, callback2 = () => {}) {
```

Comment:
This should be method of Toolbar component. callback1 and 2 should be taken from props.